exporter/lokiexporter/exporter_test.go

```diff
@@ -57,9 +59,29 @@ func TestPushLogData(t *testing.T) {
 			expectedLabel: `{exporter="OTLP", host_name="guarana"}`,
 			expectedLine:  `{"traceid":"01020304000000000000000000000000","resources":{"region.az":"eu-west-1a"}}`,
 		},
+		{
+			desc: "with resource to label and regular resource and feature gate flag exporter.loki.sendWithResourceInJSONFormat is set to true",
+			res: map[string]interface{}{
+				"host.name": "guarana",
+				"region.az": "eu-west-1a",
+			},
+			hints: map[string]interface{}{
+				"loki.resource.labels": "host.name",
+			},
+			expectedLabel:                        `{exporter="OTLP", host_name="guarana"}`,
+			expectedLine:                         `{"traceid":"01020304000000000000000000000000","resource":{"region.az":"eu-west-1a"}}`,
```

Comment:
This is the type of change that can drive users crazy. I think it would be worth adding a mention to the readme, stating that "starting from the version X, users should prepare to use 'resource' instead of 'resources', to better align with the other signals as seen in Tempo or Prometheus/Mimir".

Reply:
Thanks! added a sentence to readme